spec/models/distribution_spec.rb

```diff
@@ -67,32 +67,28 @@
   end
 
   context "Methods >" do
-    before(:each) do
-      @distribution = create(:distribution)
-      @first = create(:item, name: "AAA", category: "Foo")
-      @last = create(:item, name: "ZZZ", category: "Bar")
-    end
+    let(:distribution) { create(:distribution) }
+    let(:item) { create(:item, name: "AAA", category: "Foo") }
+    let(:donation) { create(:donation) }
```

Comment:
Switched to lazy-evaluation variables so object creation only occurs if needed for a given spec.
Read more: https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let